coriolis/tasks/replica_tasks.py

```diff
@@ -221,11 +216,11 @@ def run(self, ctxt, instance, origin, destination, task_info,
             migr_connection_info = base.marshal_migr_conn_info(
                 migr_connection_info)
             schemas.validate_value(
-                migr_connection_info,
-                schemas.CORIOLIS_DISK_SYNC_RESOURCES_CONN_INFO_SCHEMA,
-                # NOTE: we avoid raising so that the cleanup task
-                # can [try] to deal with the temporary resources.
-                raise_on_error=False)
+               migr_connection_info,
```

Comment:
Any reason for the reduced indent here? The whole rest of the file (and everything else in the repo except for the JSON schemas) seems to be using 4.

Reply:
Comment/Uncomment command in VScode helped with this one. Comments were added during testing in vim, which was diff from what VSCode does. Fixed.